packages/types/Calendar.d.ts

```diff
@@ -105,6 +105,13 @@ export interface RecurringEvent {
   tzid?: string | undefined;
 }
 
+export interface BookingLimit {
```

Comment:
nit: To make code more DRY
```suggestion
export type BookingLimit = z.infer<typeof bookingLimitsType>;
```